repo-logs-last-updated

```diff
@@ -3,7 +3,7 @@
 # reports age of oldest update-repo log for each repo* server
 
 repo_log_timestamps () {
-  wget -qO- "http://${1}.grid.iu.edu/web/log/?C=M;O=A" |
+  wget -qO- "http://${1}.opensciencegrid.org/web/log/?C=M;O=A" |
```

Comment:
Hey @brianhlin, note that `repo-logs-last-updated` viewed the `/web/log/` dir listing on the old `repo{1,2,-itb}.grid.iu.edu`.

The change here won't actually leave a working script, since
- `repo{1,2}.opensciencegrid.org` dont exist
- the new `repo` server apparently doesn't give a `/web/log/` listing

Also note that
- the new `repo` webserver apparently doesn't allow sorting by timestamp (might break the `repo_log_timestamps` function?)
- i don't know if the html format for the new `repo` server is similar enough or not to work with the regex in `repo_log_timestamps`

That being said, it seems to me that perhaps the old version of this script can just be moved to `Attic` (for future reference), rather than trying to fix it.